e2e/deploy/vault/vault.yaml

```diff
@@ -168,7 +168,7 @@ spec:
             MYSQL_ROOT_PASSWORD: s3cr3t
             MYSQL_PASSWORD: 3xtr3ms3cr3t
 
-  vaultEnvsConfig:
+  secretInitsConfig:
     - name: VAULT_LOG_LEVEL
       value: debug
     - name: VAULT_STORAGE_FILE
```

Comment:
Anyone know what this is?
@ramizpolic 
@akijakya 
There is no such env-var handled in Secret Init as far as I know.